[2025-02-11T15:40:27.468+0600] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-11T15:40:27.479+0600] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: e2e_data_loading_to_pg_dag.load_df_task manual__2025-02-11T09:40:14.817231+00:00 [queued]>
[2025-02-11T15:40:27.488+0600] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: e2e_data_loading_to_pg_dag.load_df_task manual__2025-02-11T09:40:14.817231+00:00 [queued]>
[2025-02-11T15:40:27.489+0600] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-11T15:40:27.505+0600] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): load_df_task> on 2025-02-11 09:40:14.817231+00:00
[2025-02-11T15:40:27.517+0600] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'e2e_data_loading_to_pg_dag', 'load_df_task', 'manual__2025-02-11T09:40:14.817231+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/e_to_e_processing.py', '--cfg-path', '/tmp/tmpiqeri6zc']
[2025-02-11T15:40:27.519+0600] {standard_task_runner.py:105} INFO - Job 28: Subtask load_df_task
[2025-02-11T15:40:27.522+0600] {logging_mixin.py:190} WARNING - /home/shamim/airflow_git/Airflow_project/air_venv/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=94267) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-11T15:40:27.523+0600] {standard_task_runner.py:72} INFO - Started process 94288 to run task
[2025-02-11T15:40:27.565+0600] {task_command.py:467} INFO - Running <TaskInstance: e2e_data_loading_to_pg_dag.load_df_task manual__2025-02-11T09:40:14.817231+00:00 [running]> on host shamim-X541UV
[2025-02-11T15:40:27.705+0600] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='e2e_data_loading_to_pg_dag' AIRFLOW_CTX_TASK_ID='load_df_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-11T09:40:14.817231+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-11T09:40:14.817231+00:00'
[2025-02-11T15:40:27.708+0600] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-11T15:40:27.708+0600] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-11T15:40:27.709+0600] {logging_mixin.py:190} INFO - Current task name:load_df_task state:running start_date:2025-02-11 09:40:27.479912+00:00
[2025-02-11T15:40:27.709+0600] {logging_mixin.py:190} INFO - Dag name:e2e_data_loading_to_pg_dag and current dag run status:running
[2025-02-11T15:40:27.710+0600] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-11T15:40:27.712+0600] {logging_mixin.py:190} INFO - insert into staging_table(Name,Age,Month,Salary) values %s
[2025-02-11T15:40:27.717+0600] {logging_mixin.py:190} INFO - Existing records from staging_table deleted successfully!
[2025-02-11T15:40:27.724+0600] {logging_mixin.py:190} INFO - Dataframe is inserted successfully!
[2025-02-11T15:40:27.724+0600] {logging_mixin.py:190} INFO - Insert Time: 0.012576580047607422 seconds.
[2025-02-11T15:40:27.724+0600] {python.py:240} INFO - Done. Returned value was: None
[2025-02-11T15:40:27.734+0600] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-11T15:40:27.735+0600] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=e2e_data_loading_to_pg_dag, task_id=load_df_task, run_id=manual__2025-02-11T09:40:14.817231+00:00, execution_date=20250211T094014, start_date=20250211T094027, end_date=20250211T094027
[2025-02-11T15:40:27.747+0600] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-11T15:40:27.747+0600] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-11T15:40:27.748+0600] {logging_mixin.py:190} INFO - Dag name:e2e_data_loading_to_pg_dag queued_at:2025-02-11 09:40:14.839477+00:00
[2025-02-11T15:40:27.748+0600] {logging_mixin.py:190} INFO - Task hostname:shamim-X541UV operator:PythonOperator
[2025-02-11T15:40:27.780+0600] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-11T15:40:27.804+0600] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-11T15:40:27.807+0600] {local_task_job_runner.py:245} INFO - ::endgroup::
